src/reformatters/dwd/icon_eu/forecast/template_config.py

```diff
@@ -0,0 +1,593 @@
+                encoding=Encoding(
+                    dtype="int64",
+                    fill_value=-1,
+                    compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+                    units="seconds",
+                    chunks=append_dim_coordinate_chunk_size,
+                    shards=None,
+                ),
+                attrs=CoordinateAttrs(
+                    units="seconds",
+                    statistics_approximate=StatisticsApproximate(
+                        min=str(dim_coords["lead_time"].min()),
+                        max=str(dim_coords["lead_time"].max()),
+                    ),
+                ),
+            ),
+            Coordinate(
+                name="spatial_ref",
+                encoding=Encoding(
+                    dtype="int64",
+                    fill_value=0,
+                    chunks=(),  # Scalar coordinate
+                    shards=None,
+                ),
+                attrs=CoordinateAttrs(
+                    units=None,
+                    statistics_approximate=None,
+                    # Derived by installing xarray, cfgrib, and rioxarray, and then running:
+                    #     from pyproj import CRS
+                    #     spherical_crs = CRS.from_wkt(WKT_STRING_EXTRACTED_FROM_ICON_EU_GRIB_BY_GDALINFO)
+                    #     ds = xr.load_dataset(ICON_EU_GRIB_FILENAME_FROM_DWD, engine='cfgrib')
+                    #     ds.rio.write_crs(spherical_crs)["spatial_ref"].attrs
+                    crs_wkt='GEOGCS["Coordinate System imported from GRIB file",DATUM["unnamed",SPHEROID["Sphere",6371229,0]],PRIMEM["Greenwich",0],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AXIS["Latitude",NORTH],AXIS["Longitude",EAST]]',
+                    semi_major_axis=6371229.0,
+                    semi_minor_axis=6371229.0,
+                    inverse_flattening=0.0,
+                    reference_ellipsoid_name="Sphere",
+                    longitude_of_prime_meridian=0.0,
+                    prime_meridian_name="Greenwich",
+                    geographic_crs_name="Coordinate System imported from GRIB file",
+                    horizontal_datum_name="unnamed",
+                    grid_mapping_name="latitude_longitude",
+                    spatial_ref='GEOGCS["Coordinate System imported from GRIB file",DATUM["unnamed",SPHEROID["Sphere",6371229,0]],PRIMEM["Greenwich",0],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AXIS["Latitude",NORTH],AXIS["Longitude",EAST]]',
+                    comment="From the WKT string output by gdalinfo.",
+                ),
+            ),
+        ]
+
+    @computed_field  # type: ignore[prop-decorator]
+    @property
+    def data_vars(self) -> Sequence[DwdIconEuDataVar]:
+        """Define metadata and encoding for each data variable."""
+        var_chunks: dict[Dim, int] = {
+            "init_time": 1,
+            "lead_time": 120,
+            "latitude": 73,
```

Comment:
```suggestion
            "latitude": 165,
```

The exactly evenly divisible property of 73 is nice, but assuming compression to about 20% of original size of the 4 byte float32s that will give us chunks of 830KB (`(93*73*153) * 4 * 0.2`) which is a bit small. It's also nice to have roughly square chunks geographically. 165 would give us ~1.9MB chunks and the last chunk would have data in all but the last 3 pixels.

Reply:
This sounds good to me but unfortunately both tests fail in `tests/dwd/icon_eu/forecast/template_config_test.py` because the Pydantic model defined in `reformatters.common.config_models.Encoding` [checks](https://github.com/JackKelly/reformatters/blob/416748a8be61acf4b2bb4bf85779694645b8520b/src/reformatters/common/config_models.py#L102-L108) that each shard size is an exact multiple of each chunk size! So I'll change the chunk latitude size to 219 (657 / 3) if that's OK?